ee-9/common/src/main/resources/modules/system/layers/base/org/jboss/ws/cxf/jbossws-cxf-client/main/module.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  ~ JBoss, Home of Professional Open Source.
+  ~ Copyright 2011, Red Hat, Inc., and individual contributors
+  ~ as indicated by the @author tags. See the copyright.txt file in the
+  ~ distribution for a full listing of individual contributors.
+  ~
+  ~ This is free software; you can redistribute it and/or modify it
+  ~ under the terms of the GNU Lesser General Public License as
+  ~ published by the Free Software Foundation; either version 2.1 of
+  ~ the License, or (at your option) any later version.
+  ~
+  ~ This software is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+  ~ Lesser General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU Lesser General Public
+  ~ License along with this software; if not, write to the Free
+  ~ Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+  ~ 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+  -->
+
+<module xmlns="urn:jboss:module:1.9" name="org.jboss.ws.cxf.jbossws-cxf-client">
+
+    <dependencies>
+        <!-- JAXB API + REF IMPL -->
+        <module name="javax.xml.bind.api" export="true"/>
+        <module name="com.sun.xml.bind" export="true" services="export"/>
+        <module name="javax.xml.ws.api" export="true"/>
+        <module name="javax.jws.api" export="true"/>
+        <!-- WSDL4J API -->
+        <module name="javax.wsdl4j.api" export="true"/>
+        <!-- JBossWS API -->
+        <module name="org.jboss.ws.api" export="true" />
+        <!--  JBossWS JAXWS client -->
+        <module name="org.jboss.ws.jaxws-client" export="true" services="export" />
+        <!--  JBossWS configuration of Apache CXF -->
+        <module name="org.jboss.ws.cxf.jbossws-cxf-factories" services="export" />
+        <!-- Apache CXF APIs only -->
+        <module name="org.apache.cxf" export="true" />
+        <module name="org.jboss.ws.cxf.jbossws-cxf-transports-undertow" export="true" services="export" />
+        <module name="org.jboss.ws.cxf.jbossws-cxf-transports-udp" export="true" services="export" />
+        <module name="org.jboss.jaxbintros" export="true"/>
+        <module name="javax.security.auth.message.api" export="true"/>
+        <module name="org.apache.ws.security" export="true"/>
+        <module name="org.eclipse.angus.activation" export="true" services="export"/>
```

Comment:
@yersan FYI.  The addition of this line is why this module.xml was forked.

Reply:
The activation services and classes are made available to this module by `javax.xml.bind.api `(and alias of `jakarta.xml.bind.api` which is already [exporting them](https://github.com/wildfly/wildfly/blob/e34404128a874807bb1a9fe44a441a3289ad51bd/ee-9/ee-9-api/src/main/resources/modules/system/layers/base/jakarta/xml/bind/api/main/module.xml#L37))

I understand @jimma needs to re-export them here for any other interested artifacts that depend on `org.jboss.ws.cxf.jbossws-cxf-client`. If that's not the case, I think this dependency here is unedded.

Reply:
@yersan Yes.  I added this export to get auguas activation impl accessible through the jbossws-cxf-client module dependency like what EE8 does.  But it still needs more export : https://github.com/jbossws/jbossws-cxf/commit/84d9912414e81deafde53ba9a4e33ff55a15cf15#diff- 180d1643b811361c5adef670f3808f4ad6ea8b94a04a6ced0a5be6548ff89204R89
because the jaf api issue which I tried to fix with PR : https://github.com/eclipse-ee4j/jaf/pull/91